Python/Product/VsPyProf/python.h

```diff
@@ -231,15 +233,47 @@ class PyCodeObject37 : public PyObject {
     void *co_zombieframe;       /* for optimization only (see frameobject.c) */
 
     static bool IsFor(int majorVersion, int minorVersion) {
-        return majorVersion == 3 && minorVersion >= 7;
+        return majorVersion == 3 && minorVersion == 7;
     }
 
     static bool IsFor(PythonVersion version) {
-        return version >= PythonVersion_37;
+        return version == PythonVersion_37;
+    }
+};
+
+// 3.8
+class PyCodeObject38 : public PyObject {
+public:
+    int co_argcount;            /* #arguments, except *args */
+    int co_posonlyargcount;     /* #positional only arguments */
+    int co_kwonlyargcount;      /* #keyword only arguments */
+    int co_nlocals;             /* #local variables */
+    int co_stacksize;           /* #entries needed for evaluation stack */
+    int co_flags;               /* CO_..., see below */
+    int co_firstlineno;         /* first source line number */
```

Comment:
shouldn't we initialize all these pointers below to null?

Reply:
in PythonApi.cpp I see null checking but these could be pointing to garbage

  argNames = (PyTupleObject*)((PyCodeObject25_27*)codeObj)->co_varnames;

if (argCount != 0 && argNames && argNames->ob_type == PyTuple_Type) {

Reply:
no, we never instantiate this, we only need the structure of the class so we can offset correctly into existing objects in memory

Reply:
or is there some kind of special allocator for these?